packages/components/src/generic_modal/generic_modal.tsx

```diff
@@ -31,10 +33,17 @@ export type Props = {
     backdrop?: boolean;
     backdropClassName?: string;
     children: React.ReactNode;
+
+    /**
+     * This is temporary fix until we update underlying package of generic modal.
+     * @caution Use this with enforceFocus={false}
+     */
+    shouldFocusTrapHandleFocus?: boolean;
```

Comment:
Instead of making it so that people using this also need to set `enforceFocus={false}`, could we instead make  it so that this component automatically passes `enforceFocus={false}` into the React Boostrap `Modal` automatically when this is set to true?

Reply:
Alternatively, why not always have this enabled whenever `enforceFocus` is true (or unset since it defaults to true)? Are there any cases that we wouldn't want this behaviour?

Reply:
thats a good idea, made the change

Reply:
yes thats a seems like a good idea, i have used existing prop `enforceFocus` to activate the focus trap